### PR TITLE
NazarYakh_change_language_test

### PR DESCRIPTION
**Pull request by NazarYakh**

### CODE REVIEW

File: src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    @Test
+    public void verifyUserCanChangeLanguage() {
+        new HomePage().open().header.changeLanguage("UA");
+
+        Assert.assertEquals(new HomePage()
+                .getGreetingsText(), "Ласкаво просимо!", "Language wasn't changed");
+
+        new HomePage().open().header.changeLanguage("RU");
+
+        Assert.assertEquals(new HomePage()
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java:19]: this is bad design, you re-open page each time and create 4(!) homePage obejcts

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @Test
+    public void verifyUserCanChangeLanguage() {
+        HomePage homePage = new HomePage().open();
+
+        homePage.getHeader().changeLanguage("UA");
+
+        Assert.assertEquals(homePage
+                .getGreetingsText(), "Ласкаво просимо!", "Language wasn't changed");
+
+        homePage.getHeader().changeLanguage("RU");
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java:19]: why call header twice?

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/ss/ita/rozetka/ui/elements/Header.java [hunk 3]
```diff
@@ -30,7 +30,7 @@ public ProductTypePage search() {
     }
 
     public Header changeLanguage(String language) {
```
Comment (toleksyn) [inline src/main/java/com/ss/ita/rozetka/ui/elements/Header.java:32]: step?

File: src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+public class ChangeLanguageTest extends TestRunner {
+
+    @Test
+    public void verifyUserCanChangeLanguage() {
+        HomePage homePage = new HomePage().open();
+        Header header = homePage.getHeader();
+
+        header.changeLanguage("UA");
+
+        Assert.assertEquals(homePage
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java:18]: assertj ?

File: src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+                .getGreetingsText(), "Ласкаво просимо!", "Language wasn't changed");
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java:19]: should

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+        header.changeLanguage("UA");
+
+        assertThat(homePage
+                .getGreetingsText()).as("Language should be changed").contains("Ласкаво просимо!");
+
+        header.changeLanguage("RU");
+
+        assertThat(homePage
+                .getGreetingsText()).as("Language should be changed").contains("Добро пожаловать!");
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/ChangeLanguageTest.java:26]: format things porperly

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/ss/ita/rozetka/ui/LocalizationTests.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
+package com.ss.ita.rozetka.ui;
+
+import com.ss.ita.rozetka.ui.TestUtils.TestRunner;
+import com.ss.ita.rozetka.ui.elements.Header;
+import com.ss.ita.rozetka.ui.pages.HomePage;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LocalizationTests extends TestRunner {
```
Comment (toleksyn) [inline src/test/java/com/ss/ita/rozetka/ui/LocalizationTests.java:10]: Test, singular

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Review by NyxLex — APPROVED** [review]

**Pull request merged**